lib/organization/Widgets/mydrawer.dart

```diff
@@ -1,56 +1,142 @@
-          return const Divider(height: 4,thickness: 1,color:Color.fromRGBO(146, 183, 192, 0.8),);
-              })),
-               itemCount: entries.length),
-          ),
-        ),
-      ],
-    ),
-    );
-    
-  }
-}
\ No newline at end of file
+import 'package:cura/individual/my_donations.dart';
+import 'package:cura/individual/support_team_page.dart';
+import 'package:cura/startup_screens/login.dart';
+import 'package:flutter/material.dart';
+
+import '../../individual/account_setup.dart';
+
+class mydrawer extends StatefulWidget {
+  const mydrawer({Key? key}) : super(key: key);
+
+  @override
+  State<mydrawer> createState() => _mydrawerState();
+}
+
+class _mydrawerState extends State<mydrawer> {
+  @override
+  final List<String> entries = [
+    'Doner Contact',
+    'Donation Records',
+    'Work Notifications',
+    'Post Engangements',
+    'Edit Profile',
+    'Support Team',
+    'Log Out'
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Drawer(
+      backgroundColor: const Color.fromARGB(255, 166, 223, 250),
+      child: Container(
+        decoration: const BoxDecoration(
+            gradient: LinearGradient(
+          colors: [
+            Color(0xFF6CAFB4),
+            Colors.white,
+            Colors.white,
+            Color(0xFF6CAFB4),
+          ],
+          begin: Alignment.topCenter,
+          end: Alignment.bottomCenter,
+          tileMode: TileMode.clamp,
+        )),
+        child: ListView(
+          shrinkWrap: true,
+          physics: ClampingScrollPhysics(),
+          children: [
+            Container(
+              padding: const EdgeInsets.only(top: 37, left: 16),
+              child: Column(
+                children: const [
+                  //profile image
+                  SizedBox(
+                    height: 104,
+                    width: 104,
+                    child: CircleAvatar(
+                      backgroundColor: Color.fromARGB(255, 240, 186, 156),
+                    ),
+                  ),
+                  SizedBox(
+                    height: 11,
+                  ),
+                  Text(
+                    "Welcome!\nOrganisation Name",
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                        fontSize: 20,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.black),
+                  )
+                ],
+              ),
+            ),
+            const SizedBox(
+              height: 41,
+            ),
+
+            //body
+            Expanded(
+              child: SizedBox(
+                height: double.maxFinite,
+                child: ListView.separated(
+                    itemBuilder: ((context, index) {
+                      return ListTile(
+                        onTap: () {
+                          if (entries[index] == 'Donation Records') {
+                            Navigator.push(

```

Comment:
Please use **namedRoutes** for routing, so as to stay consistent with the overall routing flow in the application.
For reference, see the RouteGenerator.dart file in /lib/shared/services/RouteGenerator.dart